src/Microsoft.AspNet.Razor/Generator/GeneratedTagHelperContext.cs

```diff
@@ -35,6 +35,8 @@ public GeneratedTagHelperContext()
             RunnerTypeName = "TagHelperRunner";
             ScopeManagerTypeName = "TagHelperScopeManager";
             ExecutionContextTypeName = "TagHelperExecutionContext";
+            HtmlEncoderPropertyName = "HtmlEncoder";
+            HtmlHelperPropertyName = "Html";
```

Comment:
Completely agree, 1 configuration value is best here, Razor doesn't need to know about the Html helper